build/dev/probe_local_instance.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+echo "Re/Create probe_local_instance_test_result file"
+RESULTFILE="$(pwd)/probe_local_instance_test_result"
```

Comment:
Let's add a [.gitignore](https://git-scm.com/docs/gitignore) file to this folder that includes the name of this result file so someone won't accidentally commit it to the repository